### ISSUE TITLE
pbench dashboard search button not working, "cannot read property 'match' of undefined"

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#1154]
The "search" button on the landing page does not appear to be working:

```
Uncaught TypeError: Cannot read property 'match' of undefined
    at umi.js:1
    at Array.map (<anonymous>)
    at onSearch (umi.js:1)
    at n.handleClick (umi.js:1)
    at Object.s (umi.js:1)
    at p (umi.js:1)
    at A (umi.js:1)
    at S (umi.js:1)
    at T (umi.js:1)
    at B (umi.js:1)
```

![PbenchSearchNotWorking-20190402](https://user-images.githubusercontent.com/949097/55422633-12a11c00-554a-11e9-991b-ba2b481f44f0.png)

**Comment by gurbirkalsi** [comment]
@portante This issue was resolved with the first phase of PR #1130

**Comment by portante** [comment]
Fixed by #1130.
